modules/codegen/src/smithy4s/codegen/Renderer.scala

```diff
@@ -943,18 +985,8 @@ private[codegen] class Renderer(compilationUnit: CompilationUnit) { self =>
       case Primitive.Int        => t => line"${t.toString}"
       case Primitive.Short      => t => line"${t.toString}"
       case Primitive.Bool       => t => line"${t.toString}"
-      case Primitive.Uuid => uuid => line"java.util.UUID.fromString($uuid)"
-      case Primitive.String => { raw =>
-        import scala.reflect.runtime.universe._
-        val str = Literal(Constant(raw))
-          .toString()
-          // Replace sequences like "\\uD83D" (how Smithy specs refer to unicode characters)
-          // with unicode character escapes like "\uD83D" that can be parsed in the regex implementations on all platforms.
-          // See https://github.com/disneystreaming/smithy4s/pull/499
-          .replace("\\\\u", "\\u")
-
-        line"$str"
-      }
+      case Primitive.Uuid   => uuid => line"java.util.UUID.fromString($uuid)"
```

Comment:
random Q: shouldn't uuids be refinements now?

Reply:
It's such a commonly used type that it warrants being first class, in particular because of optimisation at the json parsing level